tmux.conf

```diff
@@ -104,4 +104,5 @@ bind B set -g prefix ^b
 bind A set -g prefix ^a
 
 # set tmuxline snapshot in vim
-source-file ~/.tmuxline.conf
+if '[ -f ~/.tmuxline.conf.local ]' 'source-file ~/.tmuxline.conf.local'
+if '[ ! -f ~/.tmuxline.conf.local ] && [ -f ~/.tmuxline.conf ]' 'source-file ~/.tmuxline.conf'
```

Comment:
I suppoed that the `[ -f ~/.tmuxline.conf ]` check isn't necessary since that is done by the setup. Thoughts?